plugins/usbdmx/EurolitePro.cpp

```diff
@@ -176,6 +178,26 @@ bool SynchronousEurolitePro::Init() {
     return false;
   }
 
+  // USB-DMX512-PRO MK2: set baudrate to 250000
+  if (m_is_mk2) {
+    uint16_t divisor = 12;  // = 3000000 / 250000
+    uint16_t value = divisor;  // divisor & 0xFFFF
+    uint16_t index = (divisor >> 8) & 0xFF00;
+    int err = m_adaptor->ControlTransfer(
+        usb_handle,
+        LIBUSB_REQUEST_TYPE_VENDOR | LIBUSB_RECIPIENT_DEVICE |
+        LIBUSB_ENDPOINT_OUT,  // bmRequestType
+        MK2_SET_BAUD_RATE,  // bRequest
+        value,  // wValue
+        index,  // wIndex
+        NULL,  // data
+        0,  // wLength
+        500);  // timeout
```

Comment:
This timeout could be made a constant and then reused in both functions.

Reply:
Agree, done.